game/notes.cc

```diff
@@ -44,6 +45,17 @@ double Note::scoreMultiplier() const {
 double Note::powerFactor(double note) const {
 	if (type == FREESTYLE) return 1.0;
 	double error = std::abs(diff(note));
+	switch(config["game/difficulty"].i()){
+		case 0: // normal mode (original)
+			//no points if error> 150 cents, perfect score if error <50 cents
+			return clamp(1.5 - error, 0.0, 1.0);
+		case 1: // "harder mode"
+			//no points if error> 50 cents, perfect score if error < 21.51 cents (syntonic comma)
+			return clamp((0.5-error)/(0.5-0.2151), 0.0, 1.0);
+		case 2: // "perfect mode"
+			//no points if error>21.51 cents, perfect score if error < 6 cents
+			return clamp((0.2151-error)/(0.2151-0.06), 0.0, 1.0); 
+	}
 	return clamp(1.5 - error, 0.0, 1.0);
```

Comment:
Add & merge this to the switch case as default case. This gets rid of the `case 0:` block